CDCgov/prime-reportstream
# Removed old RawSubmission code

This PR removes old RawSubmission code minus the convert code because the submission was still set up to send the RawSubmission message. This has also been fixed in this PR

Test Steps:
1. Make sure the tests pass.
2. Run a message through the pipeline and make sure there are not errors.

## Changes
- Removed RawSubmission code and related tests that are no longer used. 
- Fixed broken tests
- Changed submission to not use RawSubmission message anymore. 

## Checklist

### Testing
- [ x] Tested locally?

### Process

## Linked Issues
#12428 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
